lib/bundler/gemfile.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+module Bundler
+  class Gemfile
```

Comment:
I don't think `GemfileRewriter`would suit because we are not changing gemfile anywhere. We are extracting things out of it. Need to think of something else :thinking: